tasks/task_validate.wdl

```diff
@@ -0,0 +1,52 @@
+version 1.0
+
+task export_two_tsvs {
+
+  input {
+    String        terra_project
+    String        terra_workspace
+    String        datatable1
+    String        datatable2
+  }
+  command {
+    python3 /scripts/export_large_tsv/export_large_tsv.py --project ~{terra_project} --workspace ~{terra_workspace} --entity_type ~{datatable1} --tsv_filename ~{datatable1}
+
+    python3 /scripts/export_large_tsv/export_large_tsv.py --project ~{terra_project} --workspace ~{terra_workspace} --entity_type ~{datatable2} --tsv_filename ~{datatable2}
+  }
+  runtime {
+      docker: "broadinstitute/terra-tools:tqdm"
+      memory: "1 GB"
```

Comment:
May want to make optional variable in the future to enable comparison of larger data tables

Reply:
Will do this in next PR. Good call!

- [ ] Add optional input to define runtime mem size